src/posthog-core.ts

```diff
@@ -989,6 +994,12 @@ export class PostHog {
             properties['$duration'] = parseFloat((duration_in_ms / 1000).toFixed(3))
         }
 
+        if (userAgent) {
+            properties['$browser_type'] = _isBlockedUA(userAgent, this.config.custom_blocked_useragents)
+                ? 'bot'
+                : 'browser'
+        }
```

Comment:
hmm the only thing here is now for most people every event gets $browser_type = browser because anything that is a bot isn't emitted.

I wonder if this should only be aded when filtering is inactive... I guess that's the only time it's useful

Reply:
(I may be overthinking things 🤣)

Reply:
@pauldambra Definitely can make that change. Updated - thanks for the suggestion.